File: securescope-backend/securescope-backend/src/main/java/com/securescope/scanner/dto/FindingResult.java
package com.securescope.scanner.dto;

import com.securescope.common.enums.FindingCategory;
import com.securescope.common.enums.Severity;

public record FindingResult(
	String title,
	String description,
	Severity severity,
	FindingCategory category,
	String owaspCategory,
	String filePath,
	Integer lineNumber,
	String evidence,
	String recommendation
) {
}

File: securescope-frontend/src/app/core/models/finding-result.ts
export interface FindingResult {
  title: string;
  description: string;
  severity: 'INFO' | 'LOW' | 'MEDIUM' | 'HIGH' | 'CRITICAL';
  category: string;
  owaspCategory: string;
  filePath: string;
  lineNumber: number;
  evidence: string;
  recommendation: string;
}
